lz-string/index.d.ts

```diff
@@ -57,5 +57,35 @@ declare namespace LZString {
         * @param compressed A string obtained from a call to compressToBase64().
         */
         decompressFromBase64(compressed: string): string;
+
+        /**
+        * produces ASCII strings representing the original string encoded in Base64 with a few 
+        * tweaks to make these URI safe. Hence, you can send them to the server without thinking 
+        * about URL encoding them. This saves bandwidth and CPU
+        *
+        * @param uncompressed A string which should be compressed.
+        */
+        compressToEncodedURIComponent(uncompressed: string): string;
+
+        /**
+        * Decompresses "valid" input string created by the method compressToEncodedURIComponent().
+        *
+        * @param compressed A string obtained from a call to compressToEncodedURIComponent().
+        */
+        decompressFromEncodedURIComponent(compressed: string): string;
+
+        /**
+        * produces an uint8Array
+        *
+        * @param uncompressed A string which should be compressed.
+        */
+        compressToUint8Array(uncompressed: string): Uint8Array;
+
+        /**
+        * Decompresses "valid" input string created by the method compressToUint8Array().
```

Comment:
Should be: Decompresses "valid" array created by the method compressToUint8Array().